### ISSUE TITLE
Assist dialog  drains battery when screen is off

### ISSUE THREAD

**Issue by saintman23** [home-assistant/android#4211]
**Home Assistant Android app version(s):**
2024.2.2-full
**Android version(s):**
14.0
**Device model(s):**
MT2111 - OnePlus 9RT
**Home Assistant version:**
2024.2.1
**Last working Home Assistant release (if known):**
NA

**Description of problem, include YAML if issue is related to notifications:**

1. I've set up HA as default Assistant app on device. 
2. Call it holding power button.
3. Interact with Assist.
4. Push power button to turn off the screen.

The problem is that after turning screen on (after 1 min or after 2 hours) Assist dialog is still there, draining battery all the time the screen was turned off. 
If I manually swipe it down before turning screen off, the problem is gone.

I think it is better to unload Assist dialog from memory (or sleep it, or whatever) when screen gets turned off.

**Companion App Logs:**

```

```

**Screenshot or video of problem:**

**Additional information:**

**Comment by jpelgrom** [comment]
> I think it is better to unload Assist dialog from memory (or sleep it, or whatever) when screen gets turned off.

The app is actually stopping all Assist features when [paused](https://github.com/home-assistant/android/blob/c69b725fd2388d662f062a68523063997e49da29/app/src/main/java/io/homeassistant/companion/android/assist/AssistViewModel.kt#L322) (so also on screen off).

> after turning screen on (after 1 min or after 2 hours) Assist dialog is still there

This is normal. Actually 'unloading' apps is something the system is supposed to do. You'll see this for any app you had open at the time of locking the screen. 

However, there shouldn't be anything using a lot of power such as active microphones or network connections draining the battery at this point, or preventing the system from dozing. Can you illustrate what you mean by battery drain? The Home Assistant app does a lot of things and you may also see usage for sensor updates, for example.

**Comment by saintman23** [comment]
Sorry for late reply. I did not manage to get evidence of battery drain. Looks like in my case it was caused by smth. else.

But I still see a privacy issue here. Yes, AFTER unlocking my lock screen all apps behave the same - the last one was used is opened. BUT, Assist dialog is shown after turning screen on and BEFORE unlocking the screen. So anyone has a potential opportunity to take your phone, read your last Assist dialog and even control your homeassistant! Despite the fact that your phone was locked by faceid, pin, etc.
